tests/Frontend/PathSpec.hs

```diff
@@ -33,6 +35,19 @@ import Network.Wai
 import Test.Hspec.Wai (get, post, shouldRespondWith)
 import qualified Test.Hspec.Wai.QuickCheck as Wai (property)
 
+isBrokenPath :: Main -> Bool
```

Comment:
Excluding non-working test cases is a bad practice. I can accept your decision if you provide more information. Based on that, we could make better test data generators.